tests/kafkatest/services/zookeeper.py

```diff
@@ -30,6 +30,7 @@
 class ZookeeperService(KafkaPathResolverMixin, Service):
     ROOT = "/mnt/zookeeper"
     DATA = os.path.join(ROOT, "data")
+    LOG4J_CONFIG = os.path.join(ROOT, "zookeeper-log4j.properties")
```

Comment:
Can we not use one of the existing log configs?

Reply:
We could potentially use the `kafka-log4j.properties` under `tests/kafkatest/services/kafka/templates` but that would require modifying ducktape to recursively check for `templates` folders
It currently only takes templates in the current package folder
https://github.com/confluentinc/ducktape/blob/2c9402bc2d277d7fd056a866136afd7a29d04e8f/ducktape/template.py#L43

I'm not sure if it's a good idea to do that since it can resolve in file conflicts and might cause more harm than good in this scenario.